internal/app/app.go

```diff
@@ -267,7 +266,7 @@ func makeHandle(mux *http.ServeMux, errorHandler page.ErrorHandler, store sesh.S
 					return
 				}
 
-				appData.SessionID = base64.StdEncoding.EncodeToString([]byte(loginSession.Sub))
+				appData.SessionID = loginSession.SessionID()
```

Comment:
I'm not sure why we haven't done this sooner, it has ended up in so many places

Reply:
Haha I've just been further sprinkling the codebase with the long form of this in my PR - good spot